src/main/java/io/github/syst3ms/skriptparser/registration/DefaultRegistration.java

```diff
@@ -141,11 +142,10 @@ public Class<? extends BigInteger> getRelativeType() {
                 })
                 .register();
 
-        registration.addType(
-                String.class,
-                "string",
-                "string@s"
-        );
+        registration.newType(String.class, "string", "string@s")
+                .literalParser(s -> s)
```

Comment:
This does not take into consideration quotes. So this will treat anything as a simple literal.
```suggestion
                .literalParser(input -> {
                    Optional<VariableString> string = VariableString.newInstanceWithQuotes(input, new ParserState(), registration.getLogger());
                    if (!string.isPresent())
                        return null;
                    return string.get().toString();
                })
```

Reply:
I would say just no literal parser as variable string is supposed to be for string